system-addon/lib/ActivityStream.jsm

```diff
@@ -81,16 +81,6 @@ const PREFS_CONFIG = new Map([
     title: "Show the Top Sites section on the New Tab page",
     value: true
   }],
-  ["telemetry", {
```

Comment:
Are we sure we want to make this a global setting for all ping centre clients? Wouldn't we want to control what activity stream does separately?